pkg/experimentalmetricmetadata/metadata.go

```diff
@@ -20,15 +20,15 @@ type ResourceID string
 // been changes to resource metadata such as Kubernetes labels.
 // An example of how this is used. Let's say we are dealing with a
 // Pod that has the following labels -
-// {"env": "test", "team": "otell", "usser": "bob"}. Now, let's say
+// {"env": "test", "team": "otell", "user_": "bob"}. Now, let's say
```

Comment:
I tend to suggest using `_` to indicate "this is intentionally not the correct token" -- otherwise people are forced to think "is this intentionally a typo or accidentally a typo?" -- and in this PR, it's quite rare that things appear to be intentionally misspelled...